docs/Operation-Architecture/Orbit_UI/Advanced_Workflows/Advanced_Workflows.rst

```diff
@@ -8,10 +8,12 @@ You can use Orbit to manage the following advanced workflows:
 * Perform `metadata searches`_
 
 .. toctree::
+   :maxdepth: 1
```

Comment:
toctree to remove

Reply:
Done.

Reply:
And undone... I made this invisible. It's necessary for tox to compile.